src/nvim/ex_cmds2.c

```diff
@@ -2788,6 +2788,43 @@ static FILE *fopen_noinh_readbin(char *filename)
   return fdopen(fd_tmp, READBIN);
 }
 
+typedef struct {
+  char_u *buf;
+  uint32_t pointer;
```

Comment:
I think we would normally call this `offset`, with type `size_t`, based on how it's used.